src/main/java/net/education/kyivstar/benchmark/LogicExecutionStarter.java

```diff
@@ -0,0 +1,43 @@
+package net.education.kyivstar.benchmark;
+
+import net.education.kyivstar.benchmark.logic.AtomicLogicTask;
+import net.education.kyivstar.benchmark.logic.SynLockLogicTask;
+import net.education.kyivstar.benchmark.logic.SynLogicTask;
+import net.education.kyivstar.benchmark.logic.WithoutLogicTask;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static net.education.kyivstar.benchmark.Utils.nanosToMillis;
```

Comment:
Cover more multithreading tools
1 ForkJoinPool
2 CompletableFuture (use Executors but instead of Callable<Void> use CompletableFuture)
3 Plain Thread
4 ThreadPoolExecutor 
5 ScheduledThreadPoolExecutor 

This approach gives you a flexible and extensible framework where you can add different multithreading strategies by extending the LogicExecutionStarter class.